llama-index-integrations/postprocessor/llama-index-postprocessor-rankllm-rerank/pyproject.toml

```diff
@@ -31,11 +31,12 @@ license = "MIT"
 name = "llama-index-postprocessor-rankllm-rerank"
 packages = [{include = "llama_index/"}]
 readme = "README.md"
-version = "0.1.0"
+version = "0.1.1"
 
 [tool.poetry.dependencies]
 python = ">=3.8.1,<4.0"
 llama-index-core = "^0.10.0"
+rank-llm = ">=0.2.8"
```

Comment:
pls check out the unit test failing due to this pkg is not found.
I don't think rank-llm is a pypi project.
you probably need to add this dependency locally
cc @logan-markewich

Reply:
oh hmm rank-llm requires python >=3.10 https://pypi.org/project/rank-llm/